test/Concurrency/Runtime/mainactor.swift

```diff
@@ -77,6 +79,11 @@ actor A {
 @main struct RunIt {
   static func main() async {
     print("starting")
+    if checkIfMainQueue(expectedAnswer: true) {
```

Comment:
If we don't force Windows to be running on the main thread, then this will probably fail on Windows, or at least not be all that reliable. :(